luminance-gl/src/gl33/state.rs

```diff
@@ -37,6 +37,20 @@ impl BindingStack {
   }
 }
 
+#[derive(Debug, PartialEq, Eq)]
```

Comment:
This seemed like a reasonable place to put these internal blending state structures, but I can put them somewhere else if you'd like